### ISSUE TITLE
Installation Failed - The installer failed to create partition on disk

### ISSUE THREAD

**Issue by vinyanalista** [kamarada/Linux-Kamarada-GNOME#57]
When trying to install Linux Kamarada [15.1 RC2 Build 37.8](https://github.com/kamarada/Linux-Kamarada-GNOME/releases/tag/15.1-RC2-Build37.8) using Calamares, the installation fails right at the beginning with:

```
Installation Failed

The installer failed to create partition on disk 'ATA VBOX HARDDISK'.

==========================================================================================
Create a new partition (10.00 GiB, ext4) on ‘/dev/sda’
========================================================================================== ==========================================================================================
Job: Create new partition on device ‘/dev/sda’
========================================================================================== ==========================================================================================
Job: Create file system ‘ext4’ on partition ‘/dev/sda1’
==========================================================================================
```

![calamares-kamarada](https://user-images.githubusercontent.com/1576506/74617703-4ad32e00-510d-11ea-95d9-a670f9a0f2d1.png)

**Comment by vinyanalista** [comment]
Reported to Calamares: calamares/calamares#1312

**Comment by vinyanalista** [comment]
Solved on [15.1-Final-Build49.2](https://github.com/kamarada/Linux-Kamarada-GNOME/releases/tag/15.1-Final-Build49.2) using a workaround, which is invoking Calamares with `kdesu` instead of `xdg-su`.

openSUSE Leap 15.1 currently provides Calamares v3.2.15. According to calamares/calamares#1312, that bug was fixed in Calamares 3.2.19.
